src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -411,6 +411,9 @@ private SyntaxToken Create(in TokenInfo info, SyntaxListBuilder? leading, Syntax
                     case SyntaxKind.EndOfFileToken:
                         token = SyntaxFactory.Token(leadingNode, info.Kind, trailingNode);
                         break;
+                    case SyntaxKind.BadRazorContentToken:
```

Comment:
and just to make srue i'm on the same page.  Once the lexer produces these, they're virtually certai to become skipped tokens on real C# constructs?

Reply:
gotcha.  and i imagine it also means we can do some targeted error recovery for certain patterns in C# where we see `@:`.  sgtm.